### PR TITLE
Move tasks from python files to rst

### PR DESCRIPTION
**Pull request by katyanna**
Issue #9

### CODE REVIEW

**Comment by katyanna** [comment]
Issue #9

**Comment by mei-li** [comment]
I think it would be nice to have them all in one file

**Comment by katyanna** [comment]
So we get rid of the code and leave just the assignments?

**Comment by mei-li** [comment]
yes and we can make after code files for the solutions

**Comment by katyanna** [comment]
I'm not sure about erasing the code, I believe it helps with the understanding of the assignment (for example where should they write what, and that idea of not having to start from an empty file). If you don't think the current file is too crowded, maybe we could leave it as it is and create the new .py files just for the solutions. Maybe just erase the code for tasks 6 and 7? What do you think?

**Comment by mei-li** [comment]
Sorry misunderstanding. I was not meant to erase all the code. My idea is to have a solution for task 1 that can be used as base for task2, eg.. Also that we break the exercise explanations with showing also the point of the code it should be inserted for each step (where possible) but without having the whole file, maybe the relevant snippets only. What do you think?

**Comment by mei-li** [comment]
I added you in PyLadiesBerlin, so you can work directly with the repo instead of your form if you want

File: tasks/adventure_tasks.rst [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+    # Exercise A
+
+    def dead(why):
+        print(why, "You are dead.")
+        exit(0)
+
+    start()
+
+
+7. Now with no hands!
```
Comment (mei-li) [inline tasks/adventure_tasks.rst]: why no hands?

File: tasks/adventure_tasks.rst [hunk 2]
```diff
@@ -0,0 +1,367 @@
+Adventure Tasks
+===============
+
```
Comment (mei-li) [inline tasks/adventure_tasks.rst:4]: maybe also a Table of contents would be handy here and also a link to this from the directory readme?

**Review by mei-li — APPROVED** [review]

File: tasks/adventure_tasks.rst (hunk 1 above)
Comment (katyanna) [reply tasks/adventure_tasks.rst]: just forgot to change.

**Comment by mei-li** [comment]
Can you merge now?

**Pull request merged**